.env.production

```
@@ -1,4 +1,4 @@
-DEFAULT_ENGINE_INFOS=`[
+VITE_DEFAULT_ENGINE_INFOS=`[
```

`VITE_`をつけるとレンダラープロセスから見えるようになるので、ちょっとセキュリティ強度が下がりそうです。
まあ正直ここにレンダラープロセスに渡してしまってはいけない情報が書かれることはなさそうなのですが、一旦名称をそのままにしておくのはどうでしょう 🙇 
```suggestion
DEFAULT_ENGINE_INFOS=`[
```

（それとももしかして`VITE_`とつけないとそもそもbackground.tsからですら見えない感じでしょうか 👀 ）

---

実際に試してみたのですがおっしゃる通りでした！

確かにビルドコードの中には結構簡単に含まれちゃいますね。
セキュリティ的に問題のある物を設定してはいけない、という気持ちでやろうかなと思います！